libpf/cgroupv2.go

```diff
@@ -31,29 +43,41 @@ func LookupCgroupv2(cgrouplru *lru.SyncedLRU[PID, string], pid PID) (string, err
 	}
 	defer f.Close()
 
-	var genericCgroupv2 string
+	containerID, err := extractContainerID(f)
+	if err != nil {
+		return "", err
+	}
+
+	// Cache the container ID information.
+	// To avoid busy lookups, also empty container ID information is cached.
```

Comment:
What do we mean by 'empty container ID information' ? Aren't we always returning `errNoMatch` if no container ID is found thus never reaching this place?